### PR TITLE
Add OPUS EMEA Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/emea/emea.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                x = x.strip()
+                y = y.strip()
+                result = (
+                    sentence_counter,
+                    {
+                        "id": str(sentence_counter),
+                        "translation": {l1: x, l2: y},
+                    },
+                )
+                sentence_counter += 1
```
Comment (lhoestq) [inline datasets/emea/emea.py:119]:
```suggestion
```

**Review by lhoestq — APPROVED** [review]
Thanks ! LGTM

File: datasets/emea/emea.py (hunk 1 above)
Comment (sick78) [reply datasets/emea/emea.py]: i hope that i get paid to image

Comment (sick78) [reply datasets/emea/emea.py]: image

**Pull request merged**